web_src/js/features/common-global.js

```diff
@@ -56,9 +56,8 @@ export function initGlobalEnterQuickSubmit() {
 }
 
 export function initGlobalButtonClickOnEnter() {
-  $(document).on('keypress', '.ui.button', (e) => {
-    if (e.keyCode === 13 || e.keyCode === 32) { // enter key or space bar
-      if (e.target.nodeName === 'BUTTON') return; // button already handles space&enter correctly
+  $(document).on('keypress', '.ui.button:not(button,a)', (e) => {
```

Comment:
This change seems not right.

There are still other `<a "ui button">`

Reply:
I don't quite understand.
Yes, there are other `<a "ui button">`, and if I understand https://discord.com/channels/322538954119184384/977026554909581334/1108910674257649734 correctly, neither of them work with <kbd>Enter</kbd> at the moment.

Reply:
Then I guess the selector could be `div.ui.buttion, span.ui.buttion`, instead of using `not()`.